lib/config-generator.js

```diff
@@ -127,15 +127,15 @@ class ConfigGenerator {
                 test: /\.(png|jpg|jpeg|gif|ico|svg)$/,
                 loader: 'file-loader',
                 options: {
-                    name: `images/[name]${this.webpackConfig.useVersioning ? '.[hash]' : ''}.[ext]`,
+                    name: 'images/[name].[hash].[ext]',
```

Comment:
What do you think about using a shorter hash - like `[hash:8]`? The longer seems overkill... I should have done this originally.